WinUIGallery/ControlPages/DesignGuidance/TypographyPage.xaml

```diff
@@ -170,7 +170,7 @@
                         Grid.Column="2"
                         Foreground="{ThemeResource TextFillColorSecondaryBrush}"
                         Style="{StaticResource CaptionTextBlockStyle}"
-                        Text="Size / Line height" />
+                        Text="Size" />
```

Comment:
Why remove the text "line height "? Now there will be two numbers who both mean font size at the same time??